@nav-frontend/link-styles/src/index.css

```diff
@@ -0,0 +1,40 @@
+.navds-link {
+  color: var(--navds-color-text-link);
+  background: none;
+  text-decoration: underline;
+  cursor: pointer;
+}
+.navds-link:hover {
+  text-decoration: none;
+}
+.navds-link:focus {
+  outline: none;
+  color: var(--navds-color-text-inverse);
+  text-decoration: none;
+  background-color: var(--navds-text-focus);
+  box-shadow: var(--navds-text-shadow);
+}
+.navds-link:active {
+  outline: none;
+  color: var(--navds-color-text-inverse);
+  text-decoration: none;
+  background-color: var(--navds-text-focus);
+  box-shadow: var(--navds-text-shadow);
+}
+.navds-link svg {
```

Comment:
Jeg synes stylingen av `svg` i lenker burde være opp til brukeren.

Reply:
Er enig i de fleste tilfeller av noe slikt, men er kanskje et unntak her da bruker i de aller fleste tilfeller ønsker at ikonet skal ha samme farger og hover/focus farger som teksten. Ønsker selvfølgelig eksempler på cases hvor dette ikke tilfellet !

Reply:
Jeg er mest redd for "wtf!"-reaksjoner, hvor en uventet styling-regel plutselig kicker inn. Men jeg kan se hvordan akkurat denne casen er nyttig nok til at det kan være verdt det (jeg er fortsatt skeptisk). 

Hvis vi skal beholde dette må vi fikse litt på reglene, for etter å ha testet litt med ikoner fra `@nav-frontend/icons` ser jeg at den gjør rare ting med utseende til f.eks. Add1: 
![image](https://user-images.githubusercontent.com/1894119/101498776-c936e000-396c-11eb-8e86-a96f25a5b865.png)

```
<Link href="#">
  <span>Dette er en ikonlenke</span>
  <Add1 />
</Link>
```